### PR TITLE
DB Manager: tweaks for running Qt6

### PR DESCRIPTION
**Pull request by espinafre**
Small tweaks to running in Qt 6(removed obsolete method calls QTime().start() and QTime().elapsed(), and changed an overloaded slot (QComboBox::activated()) to a non-overloaded one (textActivated()).

### CODE REVIEW

File: python/plugins/db_manager/db_plugins/data_model.py [hunk 1]
```diff
@@ -185,8 +185,7 @@ class SqlResultModel(BaseTableModel):
     def __init__(self, db, sql, parent=None):
         self.db = db.connector
 
-        t = QTime()
-        t.start()
+        t1 = QTime().currentTime()
```
Comment (nyalldawson) [inline python/plugins/db_manager/db_plugins/data_model.py:188]: A better replacement here would be QElapsedTimer

**Comment by espinafre** [comment]
@nyalldawson done :)

**Comment by espinafre** [comment]
@nyalldawson , can you restart the clang-tidy job? It doesn't seem relevant anyway for this Python plugin change. Thanks a lot!

**Comment by nyalldawson** [comment]
@espinafre Looks like there's also use of `activated[str]` in db_manager/dlg_sql_layer_window.py, do you mind updating those too? (see https://github.com/qgis/QGIS/pull/56672)

**Comment by espinafre** [comment]
@nyalldawson , all done :)

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**